Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM registry.access.redhat.com/ubi8/python-38 AS demo
+
+COPY ./ /trestle-bot
+
+WORKDIR /trestle-bot
+
+USER root
+
+# Install dependencies
+RUN  python3.8 -m pip install --upgrade pipx \
+     && pipx install poetry==1.5.1 \
+     && poetry config virtualenvs.create false \
+     && poetry install --without tests,dev
```

Comment:
We need to ensure that the installation does not prompt for user interaction. I think `--no-interaction` needs to be added here or set environment variable `POETRY_NO_INTERACTION=1`